docs/src/main/sphinx/release/release-352.md

```diff
@@ -0,0 +1,88 @@
+# Release 352 (XX YYY 2020)
+
+## General
+
+* Add support for [`WINDOW` clause](window_clause). ({issue}`651`)
+* Add {func}`version` function. ({issue}`4627`)
+* Allow prepared statement parameters for `SHOW STATS`. ({issue}`6582`)
+* Update tzdata version to 2020d. As a result, queries can no longer reference 
+  `US/Pacific-New` zone, as it has been removed. ({issue}`6660`)
+* Improve performance of queries using {func}`rank()` window function. ({issue}`6333`)
+* Improve performance of {func}`sum` and {func}`avg` for `decimal` types. ({issue}`6951`)
+* Improve join performance. ({issue}`5981`)
+* Improve query planning time. ({issue}`6544`)
+* Fix window and streaming aggregation semantics regarding peer rows. Now peer rows are
+  not distinct from each other, while previously they had to been equal. ({issue}`6472`) 
```

Comment:
- Should this be "Fix incorrect query results when .." form? 
- Also, how a user is gonna know what a streaming aggregation is, or whether they should care?
- As we're talking about "peer rows", this seems applicable to window. Let's split this bullet into two separate things -- window case, talking about peer rows, and [streaming] aggregation case (not talking about peer rows).

Reply:
@sopel39 ?

Reply:
or @erichwang ?